packages/strapi-plugin-i18n/config/functions/migrations/field/migrateForBookshelf.js

```diff
@@ -0,0 +1,125 @@
+'use strict';
+
+const { shouldBeProcesseed, getUpdatesInfo } = require('./utils');
+
+const BATCH_SIZE = 1000;
+
+const TMP_TABLE_NAME = '__tmp__i18n_field_migration';
+
+const batchInsertInTmpTable = async (updatesInfo, trx) => {
+  const tmpEntries = [];
+  updatesInfo.forEach(({ entriesIdsToUpdate, attributesValues }) => {
+    entriesIdsToUpdate.forEach(id => {
+      tmpEntries.push({ id, ...attributesValues });
+    });
+  });
+  await trx.batchInsert(TMP_TABLE_NAME, tmpEntries, 100);
+};
+
+const batchUpdate = async (updatesInfo, trx, model) => {
+  const promises = updatesInfo.map(({ entriesIdsToUpdate, attributesValues }) =>
+    trx
+      .from(model.collectionName)
+      .update(attributesValues)
+      .whereIn('id', entriesIdsToUpdate)
+  );
+  await Promise.all(promises);
+};
+
+const updateFromTmpTable = async ({ model, trx, attributesToMigrate }) => {
+  const collectionName = model.collectionName;
+  let bindings = [];
```

Comment:
you could create the bindings in each condition to avoid pushing into an array. that would make the code clearer